<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_window</class>
 <widget class="QMainWindow" name="Main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gauze</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/GUI/GUI/Fabric_icon.png</normaloff>:/GUI/GUI/Fabric_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QLineEdit" name="material_name_line_edit_"/>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Color</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Color</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="Image_controls" name="color_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Opacity</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="Image_controls" name="opacity_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Emissive</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="Image_controls" name="emissive_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Normals</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Normals</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="Image_controls" name="normals_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>Height</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="Image_controls" name="height_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Surface</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Metallic</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="Image_controls" name="metallic_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Roughness</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="Image_controls" name="roughness_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Cavity</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="Image_controls" name="cavity_image_controls_" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="export_button_">
        <property name="toolTip">
         <string>Export</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Resources.qrc">
          <normaloff>:/GUI/GUI/Export_icon.png</normaloff>:/GUI/GUI/Export_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="file_tool_bar_">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_material_action_"/>
   <addaction name="separator"/>
   <addaction name="export_action_"/>
  </widget>
  <widget class="QStatusBar" name="status_bar_"/>
  <widget class="QToolBar" name="settings_tool_bar_">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_settings_action_"/>
  </widget>
  <action name="open_settings_action_">
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/GUI/GUI/Settings_icon.png</normaloff>:/GUI/GUI/Settings_icon.png</iconset>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
   <property name="toolTip">
    <string>Open Settings</string>
   </property>
  </action>
  <action name="new_material_action_">
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/GUI/GUI/File_icon.png</normaloff>:/GUI/GUI/File_icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create new material</string>
   </property>
  </action>
  <action name="export_action_">
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/GUI/GUI/Export_icon.png</normaloff>:/GUI/GUI/Export_icon.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export current material</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Image_controls</class>
   <extends>QWidget</extends>
   <header>Image_controls.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
